package.json

```
@@ -21,6 +21,10 @@
     "plugins": [
       "markdown"
```

```suggestion
      "markdown",
      "promise"
```
Need to add rules in eslintConfig

---

The promise linting plugin is already enabled through the standard [config](https://github.com/standard/eslint-config-standard/blob/master/src/index.ts). Were there specific rules that you wanted enabled?

For now, I removed all these extra ESLint packages because they're already being provided through `eslint-config-standard`.

The `eslint-config-standard` README notes that you can install these three ESLint configs if you want to do a custom config. This repo never implemented a custom config, so there's no need to add them since `eslint-standard-config` includes them already.